application/src/main/java/org/ehrbase/application/abac/AbacConfig.java

```diff
@@ -19,156 +19,144 @@
 package org.ehrbase.application.abac;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
-import java.io.IOException;
-import java.net.URI;
-import java.net.http.HttpRequest;
-import java.net.http.HttpRequest.BodyPublishers;
-import java.net.http.HttpResponse;
-import java.net.http.HttpResponse.BodyHandlers;
-import java.util.Map;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.ContentType;
+import org.apache.http.entity.StringEntity;
 import org.ehrbase.api.exception.InternalServerException;
-import org.ehrbase.application.config.HttpClientConfig;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
+import java.io.IOException;
+import java.net.URI;
+import java.util.Map;
+
 @ConditionalOnProperty(name = "abac.enabled")
 @Configuration
 @EnableConfigurationProperties
 @ConfigurationProperties(prefix = "abac")
+@SuppressWarnings("java:S6212")
 public class AbacConfig {
 
-  public enum AbacType {
```

Comment:
Next to swapping the HTTP client are the most changes here regarding the file formatting? That's just minor but on my machine it all gets formatted back again (I think, I didn't compare it with the original version and it might be an even different third version my IDE creates). Did you use the Google Java style we agreed on for EHRbase?

Reply:
I apologize. I try to never reformat the class that I touch to avoid this kind of changes but sometimes i I do not realize that I pressed Ctrl + Alt + O and Ctrl + Alt + L.

I have the issue and it the reason why I was asking to apply the same code convention based on a common configuration that even might be included in the project or checked using Checkstyle plugin.
The current code base is not applying the same code convetions everywhere. For instance, some classes are using 2 spaces indentation and some others 4.

PS: I am using the default coding style coming with IntelliJ which is very close to Sun(Oracle) code conventions. But I do not have any issue to switch for something else if I know which code conventions I have to use.